docs/advanced_guide/cloud/azure/notebooks/multi_node_search/multi_node_search.ipynb

```diff
@@ -0,0 +1,1719 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multi-node Search\n",
+    "\n",
+    "This notebook and accompanying code shows how to run an Archai Network Architecture Search (NAS)\n",
+    "using an Azure Machine Learning Workspace with partial training of models on a GPU cluster providing\n",
+    "validation accuracies to guide that search. This example requires a storage account and Azure\n",
+    "machine learning workspace specified in a config.json file like this:\n",
```

Comment:
Maybe we can add links to show how to get this data, e.g.:

https://learn.microsoft.com/en-us/azure/storage/common/storage-account-get-info?tabs=portal

https://learn.microsoft.com/en-us/azure/machine-learning/how-to-configure-environment#local-and-dsvm-only-create-a-workspace-configuration-file

Reply:
great idea, thanks. I pushed a fix